soql-standalone-parser/src/main/scala/com/socrata/soql/ast/Select.scala

```diff
@@ -42,7 +42,7 @@ case class JoinTable(tableName: TableName) extends JoinSelect {
 }
 case class JoinQuery(selects: BinaryTree[Select], definiteAlias: String) extends JoinSelect {
   val alias = Some(definiteAlias)
-  override def toString = "(" + Select.toString(selects) + ") AS `" + TableName.removeValidPrefix(definiteAlias) + "`"
+  override def toString = "(" + Select.toString(selects) + ") AS @" + TableName.removeValidPrefix(definiteAlias) + ""
```

Comment:
Can't decide if that ending empty string is mischief or necessary for some mysterious reason.

Reply:
Mischief!